runtimes.yml

```diff
@@ -0,0 +1,59 @@
+version: 0.1
```

Comment:
The default CodeBuild images specify a runtimes.yml file, which allows you to switch between runtimes in the buildspec configuration files. We are only installing a limited set of versions of each type of runtime, but we can extend to add support for dotnet 3 (older) and allow users to swap to that runtime in the buildspec file.

Runtimes docs: https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#runtime-versions-buildspec-file